src/main/java/com/squareup/catalog/demo/Main.java

```diff
@@ -223,8 +260,8 @@ private void executeExample(String exampleName, boolean cleanup, CatalogApi cata
           } else {
             example.execute(catalogApi, locationsApi);
           }
-        } catch (ApiException e) {
-          handleApiException(e);
+        } catch (Exception e) {
+          throw new RuntimeException(e);
```

Comment:
Could you add a comment here that explains that in a real app you would probably want to handle the exception?